options/create.go

```diff
@@ -114,9 +114,14 @@ func (opts *Create) Validate() error {
 
 	if opts.WorkingDirectory != "" && opts.isLocal() {
 		info, err := os.Stat(opts.WorkingDirectory)
-
-		if os.IsNotExist(err) {
-			catcher.Errorf("cannot use '%s' as working directory because it does not exist", opts.WorkingDirectory)
+		if err != nil {
```

Comment:
Since Go evaluates if-else if blocks in top down order and stop at the first true boolean, this can be simplified down to a series of if-else if blocks:
```go
if os.IsNotExist(err) {
    catcher.Errorf(...)
} else if err != nil {
    catcher.Wrapf(err, ...)
} else if info == nil {
    catcher.Errorf(...)
} else if !info.isDir() { // At this point, we already confirmed info is not nil
    catcher.Errorf(...)
}
```

Reply:
oh... updated